eclass/tree-sitter-grammar.eclass

```diff
@@ -40,15 +40,10 @@ EXPORT_FUNCTIONS src_compile src_install
 # @INTERNAL
 # @DESCRIPTION:
 # This internal function determines the ABI version of a grammar library based
-# on the package version.
+# on a constant in the source file.
 _get_tsg_abi_ver() {
-	if ver_test -gt 0.21; then
-		die "Grammar too new; unknown ABI version"
-	elif ver_test -ge 0.19.0; then
-		echo 13
-	else
-		die "Grammar too old; unknown ABI version"
-	fi
+	sed -n '/#define LANGUAGE_VERSION/s/.* //p' "${S}"/parser.c ||
```

Comment:
I don't understand what this sed command is doing, I could look it up, but that kind of defeats the point. Can we add a comment?